collector/cve_test.go

```diff
@@ -40,7 +43,8 @@ func TestCVEResolver(t *testing.T) {
 
 	cveResolver := collector.NewCVEResolver(authClient, witClient)
 	codebaseURL := "git@github.com:testrepo/testproject1.git"
-	recvs, _, err := cveResolver(context.Background(), codebaseURL)
+	ctx, _ := client.ContextWithRequestID(context.Background())
+	recvs, _, err := cveResolver(ctx, codebaseURL)
```

Comment:
I was wondering about the test which is verifying 500 from wit for missing requestID. Do we have it already?

Reply:
not getting you.

Reply:
Do we have some tests where WIT it returning 500 for missing header?

Reply:
now, I changed to 400.  Is this what your concern ??

Reply:
it's good now @nurali-techie